app/views/layouts/application.html.erb

```diff
@@ -1,15 +1,20 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>SampleApp</title>
+    <title><%= yield(:title) %> | Ruby on Rails Tutorial Sample App</title>
```

Comment:
` Ruby on Rails Tutorial Sample App`
I18n của tui mô (sad)

`sử dụng I18n cho TOÀN BỘ TEXT của TẤT CẢ CÁC FILE HTML nhé`

Reply:
Ruby on Rails Tutorial Sample App
I18n của tui mô (sad)

sử dụng I18n cho TOÀN BỘ TEXT của TẤT CẢ CÁC FILE HTML nhé
>> Sao ko fix gái